packages/web-components/fast-foundation/src/combobox/combobox.template.ts

```diff
@@ -14,8 +14,10 @@ export function comboboxTemplate<T extends FASTCombobox>(
         <template
             aria-disabled="${x => x.ariaDisabled}"
             autocomplete="${x => x.autocomplete}"
-            class="${x => (x.open ? "open" : "")} ${x =>
-                x.disabled ? "disabled" : ""} ${x => x.position}"
+            class="${x =>
```

Comment:
If open is necessary here we can leave it, but everything else should have an attribute, should it not?